src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -234,6 +252,213 @@ protected ActionResult execute() throws EntityDoesNotExistException {
+            appendToStatusToAdmin(feedbackResponseComment);
+        }
+    }
+
+    protected void appendToStatusToAdmin(FeedbackResponseCommentAttributes feedbackResponseComment) {
+        statusToAdmin += this.getClass().getName() + ":<br>"
+                                 + "Adding comment to response: " + feedbackResponseComment.feedbackResponseId + "<br>"
+                                 + "in course/feedback session: " + feedbackResponseComment.courseId + "/"
+                                 + feedbackResponseComment.feedbackSessionName + "<br>"
+                                 + "by: " + feedbackResponseComment.commentGiver + " at "
+                                 + feedbackResponseComment.createdAt + "<br>"
+                                 + "comment text: " + feedbackResponseComment.commentText.getValue();
+    }
+
+    // Extracts two types of feedback response comments- ones which were newly added and ones which were edited
+    private void extractFeedbackResponseCommentsForResponseData(FeedbackQuestionAttributes questionAttributes,
+                                                                int questionIndex, FeedbackResponseAttributes response,
+                                                                int responseIndex) {
+        if (!questionAttributes.getQuestionDetails().isStudentCommentsOnResponsesAllowed()) {
+            return;
+        }
+        //When comment is added, comment text parameter is of type responsecommenttext-responseIndex-1-questionIndex
+        String commentText = getRequestParamValue(Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_TEXT
+                                                          + "-" + responseIndex + "-"
+                                                          + Const.GIVER_INDEX_FOR_FEEDBACK_SUBMISSION_PAGE
+                                                          + "-" + questionIndex);
+        if (!StringHelper.isEmpty(commentText)) {
+            String commentIndex = "-" + responseIndex + "-"
+                                          + Const.GIVER_INDEX_FOR_FEEDBACK_SUBMISSION_PAGE + "-" + questionIndex;
+
+            questionIdsForComments.put(commentIndex, questionAttributes.getId());
+            responseGiverMapForComments.put(commentIndex, response.giver);
+            responseRecipientMapForComments.put(commentIndex, response.recipient);
+            commentsToAddText.put(commentIndex, commentText);
+            return;
+        }
+
+        //commentId exists only when comment is edited but not when it is newly added
+        // and it is of type responsecommentid-responseIndex-questionIndex-1
+        String commentId =
+                getRequestParamValue(Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID
+                                             + "-" + responseIndex + "-"
+                                             + Const.GIVER_INDEX_FOR_FEEDBACK_SUBMISSION_PAGE
+                                             + "-" + questionIndex + "-" + 1);
```

Comment:
It is the `frcIndex` of `feedbackResponseCommentRow.tag`
```
<c:choose>
  <c:when test="${not empty firstIndex && not empty secondIndex && not empty thirdIndex && not empty fourthIndex && not empty frcIndex}">
    <c:set var="divId" value="${fourthIndex}-${firstIndex}-${secondIndex}-${thirdIndex}-${frcIndex}" />
  </c:when>
  <c:when test="${not empty firstIndex && not empty secondIndex && not empty thirdIndex && not empty frcIndex && not empty viewType}">
    <c:set var="divId" value="${viewType}-${firstIndex}-${secondIndex}-${thirdIndex}-${frcIndex}" />
  </c:when>
  <c:when test="${not empty firstIndex && not empty secondIndex && not empty thirdIndex && not empty frcIndex}">
    <c:set var="divId" value="${firstIndex}-${secondIndex}-${thirdIndex}-${frcIndex}" />
  </c:when>
  <c:otherwise>
    <c:set var="divId" value="${frc.commentId}" />
  </c:otherwise>
</c:choose>
<c:choose>
```